src/main/cpp/CMakeLists.txt

```diff
@@ -253,3 +253,5 @@ if(SPARK_RAPIDS_JNI_BUILD_BENCHMARKS)
   rapids_cpm_nvbench()
   add_subdirectory(benchmarks)
 endif()
+
+add_subdirectory(faultinj)
```

Comment:
Maybe everything called `fault_injection` is more expressive and better than `faultinj`.

Reply:
We can reconsider, it feels too long for a directory name at the moment. It contains a README.md with enough background